packages/app/src/cli/models/app/extensions.ts

```diff
@@ -28,8 +28,19 @@ export const UIExtensionConfigurationSchema = schema.define.object({
     )
     .default([]),
   extensionPoints: schema.define.array(schema.define.string()).optional(),
-  capabilities: schema.define.any().optional(),
-
+  capabilities: schema.define
+    .object({
+      block_progress: schema.define.boolean().default(false),
+      network_access: schema.define.boolean().default(false),
+    })
+    .default({
+      block_progress: false,
+      network_access: false,
+    })
+    .transform((capabilities) => ({
+      blockProgress: capabilities.block_progress,
+      networkAccess: capabilities.network_access,
+    })),
```

Comment:
With these changes we get different JSON between go and Node for the product subscriptions JSON.

Node on the left, go on the right (I think Node is correct):

<img width="1434" alt="Screen Shot 2022-09-28 at 11 12 20 AM" src="https://user-images.githubusercontent.com/690791/192817175-73269f45-fb68-449e-b8b9-60bd998158e8.png">

This is when going to `/extensions`.

Reply:
That makes me think the attribute was not being passed correctly before. @vividviolet is the Node one accurate?

Reply:
#### `localizationStatus`
@vividviolet I aligned the behaviour with Go's

#### Empty `resource`
I aligned this one with Go too.

#### networkAccess
The changes in the PR introduced a regression in the Go flow. I fixed it too.